### DISCUSSION TITLE
EEPROM problem

### DISCUSSION THREAD

**Discussion by IPdotSetAF** [bxparks/AceRoutine#28] [Q&A]
hey.
i started using this lib for my new project and i am using it on an ESP8266 .
the problem is that when i try writing to the EEPROM it does not write although EEPROM.commit() returns true, and i get errors on the next boot and code does not run any more.

it can read the data tho , i tried saving and reading from EEPROM from a normal sketch and it worked fine.

do you have any suggestion what might be the problem?

**Comment by bxparks** [comment]
Unfortunately, you have not provided enough information for me to debug anything. Can you strip down your program to its minimal reproducible form, taking out everything unrelated to reproducing the bug, and post it here?
